site/docs/guides/admin/servers/configuring-the-view-services.md

```diff
@@ -160,25 +160,25 @@ It is possible to list the registered view services for an [OMAG Server Platform
                 "serviceName": "Glossary Author",
                 "serviceURLMarker": "glossary-author",
                 "serviceDescription": "View Service for glossary authoring.",
-                "serviceWiki": "https://odpi.github.io/egeria/open-metadata-implementation/access-services/subject-area/"
+                "serviceWiki": "https://egeria-project.org/services/omvs/glossary-author/overview/"
```

Comment:
This file is showing the output from the OMAG server platform. It needs to match the code. In the code, the links all work but they are of the form "https://odpi.github.io/egeria-docs/services/omvs/tex/overview".